src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/maui.aotprofile.txt

```diff
@@ -1661,19 +1657,15 @@ Methods:
 	Microsoft.Maui.Controls.BindableProperty Microsoft.Maui.Controls.Internals.TemplatedItemsList`2<Microsoft.Maui.Controls.MultiPage`1<Microsoft.Maui.Controls.Page>, Microsoft.Maui.Controls.Page>:get_GroupHeaderTemplateProperty ()
 	Microsoft.Maui.Controls.BindableProperty Microsoft.Maui.Controls.Internals.TemplatedItemsList`2<Microsoft.Maui.Controls.MultiPage`1<Microsoft.Maui.Controls.Page>, Microsoft.Maui.Controls.Page>:get_IsGroupingEnabledProperty ()
 	Microsoft.Maui.Controls.BindableProperty Microsoft.Maui.Controls.Internals.TemplatedItemsList`2<Microsoft.Maui.Controls.MultiPage`1<Microsoft.Maui.Controls.Page>, Microsoft.Maui.Controls.Page>:get_ProgressiveLoadingProperty ()
+	Microsoft.Maui.Controls.BindableProperty Microsoft.Maui.Controls.Routing:CreateRouteProperty ()
 	Microsoft.Maui.Controls.BindablePropertyKey Microsoft.Maui.Controls.BindableProperty:CreateAttachedReadOnly (string,System.Type,System.Type,object,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.BindableProperty/ValidateValueDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangedDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangingDelegate,Microsoft.Maui.Controls.BindableProperty/CoerceValueDelegate,Microsoft.Maui.Controls.BindableProperty/CreateDefaultValueDelegate)
 	Microsoft.Maui.Controls.BindablePropertyKey Microsoft.Maui.Controls.BindableProperty:CreateReadOnly (string,System.Type,System.Type,object,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.BindableProperty/ValidateValueDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangedDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangingDelegate,Microsoft.Maui.Controls.BindableProperty/CoerceValueDelegate,Microsoft.Maui.Controls.BindableProperty/CreateDefaultValueDelegate)
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.AppThemeBinding:Clone ()
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Binding:Clone ()
+	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.SetterSpecificityList`1<Microsoft.Maui.Controls.BindingBase>:get_Item (Microsoft.Maui.Controls.SetterSpecificity)
+	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.SetterSpecificityList`1<Microsoft.Maui.Controls.BindingBase>:GetClearedValue ()
+	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.SetterSpecificityList`1<Microsoft.Maui.Controls.BindingBase>:GetValue ()
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Xaml.AppThemeBindingExtension:Microsoft.Maui.Controls.Xaml.IMarkupExtension<Microsoft.Maui.Controls.BindingBase>.ProvideValue (System.IServiceProvider)
-	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Xaml.BindingExtension:Microsoft.Maui.Controls.Xaml.IMarkupExtension<Microsoft.Maui.Controls.BindingBase>.ProvideValue (System.IServiceProvider)
```

Comment:
So we lost the built-in profile for bindings? https://github.com/dotnet/maui/commit/018c74670ee5e14abbd2141b13140affbfa6b701 says it saved 1ms:
```
Total (ms) |  Self (ms) | Method
          0.99 |       0.99 | Microsoft.Maui.Controls.Xaml.BindingExtension:Microsoft.Maui.Controls.Xaml.IMarkupExtension<Microsoft.Maui.Controls.BindingBase>.ProvideValue (System.IServiceProvider)
```
We could make a `ViewModel` and really bind the value? Or somehow ignore XC022?